src/Components/SelectionBox/SelectionBox.jsx

```diff
@@ -55,7 +60,11 @@ const SelectionBox = ({
         </ul>
       </div>
       <div className="select-box-footer">
-        <Button block appearance="red" onClick={handleClick}>
+        <Button
```

Comment:
Shouldnt we disable the cloud button for now?

Reply:
done